### PR TITLE
BC-916 - Wrapping up rooms overview

### PR DESCRIPTION
**Pull request by pkleybolte**
# Short Description

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-916

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/store/rooms.ts [hunk 1]
```diff
@@ -154,6 +154,17 @@ export class Rooms extends VuexModule {
...
+				(room) =>
+					room.xPosition === payload.xPosition &&
+					room.yPosition === payload.yPosition
+			);
+			const roomsData = [...this.roomsData];
+			roomsData[roomIndex] = {
+				...this.roomsData[roomIndex],
+				title: payload.title,
+			};
+			this.setRoomData(roomsData);
```
Comment (pkleybolte) [inline src/store/rooms.ts:167]: Missing Test for this change

File: src/pages/rooms-overview.vue [hunk 2]
```diff
@@ -1,71 +1,82 @@
...
-		<v-row class="text-left pl-2">
-			<v-col cols="8" sm="6" md="8" lg="8">
-				<v-text-field
-					ref="search"
-					v-model="searchText"
-					:label="$t('common.words.search')"
-					:append-icon="mdiMagnify"
+	<default-wireframe ref="main" headline="" :full-width="true">
+		<template slot="header">
+			<h1 class="text-h3">Current Courses</h1>
```
Comment (pkleybolte) [inline src/pages/rooms-overview.vue:4]: Missing Translation

File: src/pages/rooms-overview.vue [hunk 3]
```diff
@@ -1,71 +1,82 @@
...
+			<div class="mb-5">
+				<v-btn color="secondary" outlined small to="/rooms-list"
+					>All Courses</v-btn
```
Comment (pkleybolte) [inline src/pages/rooms-overview.vue:7]: Missing Translation

File: src/components/atoms/vRoomAvatar.vue [hunk 4]
```diff
@@ -90,8 +90,9 @@ export default {
 			this.hovered = true;
 			this.isDragging = false;
 		},
-		dragEnd() {
+		dragend() {
 			this.isDragging = false;
+			this.$emit("dragend", this.item);
```
Comment (murat-merdoglu-hpi) [inline src/components/atoms/vRoomAvatar.vue:95]: Not necessary to send items as payload

File: src/components/molecules/vRoomGroupAvatar.vue [hunk 5]
```diff
@@ -76,29 +82,34 @@ export default {
...
 		},
 		dragEnter() {
 			this.hovered = true;
+			this.isDragging = false;
 		},
 		dropAvatar() {
 			this.$emit("drop");
 		},
+		dragend() {
+			this.$emit("dragend", this.item);
```
Comment (murat-merdoglu-hpi) [inline src/components/molecules/vRoomGroupAvatar.vue:102]: Not necessary to send items as payload

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Review by murat-merdoglu-hpi — DISMISSED** [review]

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Comment by murat-merdoglu-hpi** [comment]
@schul-cloud-bot update screenshots

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Pull request merged**